run-integration-tests.py

```diff
@@ -60,6 +60,7 @@ def run_scala_integration_tests(root_dir, version, test_name, extra_maven_repo,
 def run_python_integration_tests(root_dir, version, test_name, extra_maven_repo, extra_packages, jars, conf):
     print("\n\n##### Running Python tests on version %s #####" % str(version))
     clear_artifact_cache()
+    run_cmd(["build/sbt", "publishM2"])
```

Comment:
Note: this needs to be removed later, and changes from https://github.com/delta-io/delta/pull/990 needs to be merged.